### PR TITLE
display userlocation

### PR DESCRIPTION
**Pull request by lauravikanis**
Display a Marker with userlocation

### CODE REVIEW

**Review by mhstrkmp — COMMENTED** [review]
looks awesome 🚀

File: client/src/hooks/useCurrentLocation.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from "react";
+
+const useCurrentLocation = (options = {}) => {
+  // store location in state
+  const [location, setLocation] = useState();
+  // store error message in state
+  const [error, setError] = useState();
+
```
Comment (boldCM) [inline client/src/hooks/useCurrentLocation.js:8]: remove unneccessary comments ;)

**Review by boldCM — APPROVED** [review]
lgtm 🎉

File: client/src/hooks/useCurrentLocation.js (hunk 1 above)
Comment (lauravikanis) [reply client/src/hooks/useCurrentLocation.js:8]: we already talked about this 🤡

**Pull request merged**